fast_barcode_scanner/lib/src/camera_controller.dart

```diff
@@ -331,8 +331,28 @@ class _CameraController implements CameraController {
     }
   }
 
-  void _onDetectHandler(Barcode code) {
+  void _onDetectHandler(List<Barcode> codes) {
     events.value = ScannerEvent.detected;
-    _onScan?.call(code);
+    _onScan?.call(codes);
   }
 }
+
+class ScannedBarcodes {
```

Comment:
I think this deserves its own file.

Reply:
I suppose that's fine, though I think it's better here. `ScannedBarcodes` is really just an internal structure to couple a list of barcodes and the time at which they were scanned. This isn't a useful structure outside of this file. It's basically a typed Tuple. If it's ok with you, I'd rather leave it here. But if you insist in moving it out I don't particularly mind.